xml/telnet_banners.xml

```diff
@@ -0,0 +1,648 @@
+      24gYW4gaTY4Ng0KbG9naW46Cg==
+    </example>
+    <!-- CentOS release 6.10 (Final)\r\nKernel 2.6.32-754.2.1.el6.x86_64 on an x86_64\r\nserver-01.2 login: -->
+    <example _encoding="base64" os.arch="x86_64" host.name="server-01.2">
+      Q2VudE9TIHJlbGVhc2UgNi4xMCAoRmluYWwpDQpLZXJuZWwgMi42LjMyLTc1NC4yLjEuZWw2L
+      ng4Nl82NCBvbiBhbiB4ODZfNjQNCnNlcnZlci0wMS4yIGxvZ2luOgo=
+    </example>
+    <param pos="0" name="service.vendor" value="CentOS"/>
+    <param pos="0" name="service.product" value="telnet"/>
+    <param pos="0" name="os.vendor" value="CentOS"/>
+    <param pos="0" name="os.family" value="Linux"/>
+    <param pos="0" name="os.product" value="Linux"/>
+    <param pos="1" name="os.version"/>
+    <param pos="2" name="linux.kernel.version"/>
+    <param pos="3" name="os.arch"/>
+    <param pos="0" name="os.cpe23" value="cpe:/o:centos:centos:{os.version}"/>
+    <param pos="4" name="host.name"/>
+  </fingerprint>
+  <fingerprint pattern="^(?:\r|\n)*(RT-AC\d\d\w) login:\s*$">
+    <description>Asus Wireless Access Point/Router - RT-AC prefix</description>
+    <example hw.product="RT-AC54U">RT-AC54U login:</example>
+    <example hw.product="RT-AC68R">RT-AC68R login:</example>
+    <param pos="0" name="service.product" value="telnet"/>
+    <param pos="0" name="os.family" value="Linux"/>
+    <param pos="0" name="os.product" value="Linux"/>
+    <param pos="0" name="hw.vendor" value="Asus"/>
+    <param pos="0" name="hw.device" value="WAP"/>
+    <param pos="1" name="hw.product"/>
+  </fingerprint>
+  <fingerprint pattern="^(?:\r|\n)*(AC\d\d00) login:\s*$">
+    <description>Asus Wireless Access Point/Router - AC prefix</description>
+    <example hw.product="AC1000">AC1000 login:</example>
+    <example hw.product="AC3000">AC3000 login:</example>
+    <param pos="0" name="service.product" value="telnet"/>
+    <param pos="0" name="os.family" value="Linux"/>
+    <param pos="0" name="os.product" value="Linux"/>
+    <param pos="0" name="hw.vendor" value="Asus"/>
+    <param pos="0" name="hw.device" value="WAP"/>
+    <param pos="1" name="hw.product"/>
+  </fingerprint>
+  <fingerprint pattern="^(?:\r|\n)*(Air5\d+\w{0,2}) login:\s*$">
+    <description>Airties</description>
+    <example hw.product="Air5650">Air5650 login:</example>
+    <example hw.product="Air5650TT">Air5650TT login:</example>
+    <param pos="0" name="service.product" value="telnet"/>
+    <param pos="0" name="hw.vendor" value="Airties"/>
+    <param pos="0" name="hw.device" value="WAP"/>
+    <param pos="1" name="hw.product"/>
+  </fingerprint>
+  <fingerprint pattern="^Amazon Linux AMI release ([\d.]+)(?:\r|\n)+Kernel ([\w.-]+) on an (\w+)(?:\r|\n)+(?:([\w.-]+) )?login:\s*$">
+    <description>Amazon Linux AMI</description>
+    <!-- Amazon Linux AMI release 2013.09\r\nKernel 3.4.68-59.97.amzn1.x86_64 on an x86_64\r\nserver-01.2 login: -->
+    <example _encoding="base64" os.version="2013.09" linux.kernel.version="3.4.68-59.97.amzn1.x86_64">
```

Comment:
minor, but if possible any examples should test every extraction.  In this case `os.arch` and `host.name` need testing.  Here and maybe elsewhere.